substratevm/src/com.oracle.svm.core.genscavenge/src/com/oracle/svm/core/genscavenge/GCImpl.java

```diff
@@ -223,7 +223,7 @@ private boolean collectImpl(GCCause cause, long requestingNanoTime, boolean forc
                     }
                 }
             } finally {
-                JfrGCEventSupport.emitGCPhasePauseEvent(getCollectionEpoch(), cause.getName(), startTicks);
+                JfrGCEventSupport.emitGarbageCollectionEvent(getCollectionEpoch(), GenScavengeGCName.GenScavenge, cause, startTicks);
```

Comment:
There isn't really any need to pass the GC name as an argument as `JfrGCEventSupport` is GC-specific anyways.

Reply:
Sure